nuget/framework/nunit.nuspec

```diff
@@ -30,6 +30,8 @@ Supported platforms:
       <group targetFramework="net45" />
       <group targetFramework="netstandard1.6">
         <dependency id="NETStandard.Library" version="1.6.0" />
+        <dependency id="System.Diagnostics.Process" version="4.3.0" />
+        <dependency id="System.Xml.XmlDocument" version="4.3.0" />
```

Comment:
Oops, these are in two PRs! To copy my comments across from the other:

Both of these packages look like they're not actually supported on iOS/Android. 😞 I haven't tried it, but unzipping the packages, they have those _._ files...

We should figure this out one way or the other before merging. Adding dependencies to netstandard is what broke everything last time. 😞

Reply:
Point number 2: this is making me think more and more we should have Android and UWP CI. 😞

Reply:
I'll check into that. I'm not liking the trend, but that part is simple enough to undo.